arrow-core-data/src/main/kotlin/arrow/typeclasses/Traverse.kt

```diff
@@ -340,41 +334,36 @@ import arrow.core.ValidatedNel
  *
- * Going back to our `IO` example from the beginning with concurrency in mind, we can get an [Applicative] instance for `IO`, that traverses concurrently, by using `parApplicative`. Contrary to `IO.applicative()`, which runs in sequence.
- *
- * It is worth mentioning that `parApplicative` does not implement `lazyAp` in contrast to `IO.applicative()`, which means `parApplicative` creates all IO's upfront and executes them in parallel. More importantly, `parApplicative` breaks monad-applicative-consistency laws by design. The aforementioned law holds in the case of `IO.applicative()`, where an effect only runs , when the previous one is successful - hence the monadic nature.
- *
- * Continuing with the example, we traverse a `List<A>` with its [Traverse] instance `ListK.traverse()` and a function`(A) -> IO<B>`, we can imagine the traversal as a scatter-gather. Each `A` creates a concurrent computation that will produce a `B` (the scatter), and as the `IO` operations completes they will be gathered back into a `List`.
- *
- * Ultimately, we utilize `parTraverse` that calls traverse with `parApplicative` in an concurrent environment [F] - in the following example `IO`:
+ * Continuing with the example, we traverse a `List<A>` with its [traverse] and a function `(A) -> Either<NotFound, B>`, we can imagine the traversal as a scatter-gather. Each `A` creates a concurrent computation that will produce a `B` (the scatter), and as the suspended `Either` operations completes they will be gathered back into a `List`.
  *
  * ```kotlin:ank:playground
- * import arrow.fx.IO
- * import arrow.fx.extensions.fx
- * import arrow.fx.extensions.io.concurrent.parTraverse
- * import arrow.fx.extensions.io.unsafeRun.runBlocking
- * import arrow.unsafe
+ * import arrow.core.computations.either
+ * import arrow.fx.coroutines.parTraverse
+ * import arrow.core.Right
  *
  * interface Profile
  * interface User
  * //sampleStart
  * data class DummyUser(val name: String) : User
  * data class DummyProfile(val u: User) : Profile
  *
- * fun userInfo(u: User): IO<Profile> =
- *   IO { DummyProfile(u) } // this can be a call to the DB
+ * suspend fun userInfo(u: User): Either<NotFound, Profile> =
+ *   Right(DummyProfile(u)) // this can be a call to the DB
  *
- * fun List<User>.processLogin(): IO<List<Profile>> =
- *   parTraverse { userInfo(it) }
+ * suspend fun List<User>.processLogin(): Either<NotFound, List<Profile>> =
+ *   either {
+ *     parTraverse { userInfo(it)() }
+ *   }
  *
- * fun program(): IO<Unit> = IO.fx {
+ * suspend fun program(): Unit {
  *   val list = listOf(DummyUser("Micheal"), DummyUser("Juan"), DummyUser("T'Challa"))
- *     .processLogin()()
+ *     .processLogin()
```

Comment:
Doesn't this have to be invoked?

Reply:
Here we want to show the result of `processLogin` which is a `Either<NotFound, List<Profile>>`